test/index-spec.js

```diff
@@ -0,0 +1,21 @@
+const flattener = require('../src/index.js');
+const assert = require('assert');
+
+describe('flattener camelCase', () => {
+  it('should flatten object to camel-case style', () => {
```

Comment:
would be nice to have more test cases, objects with prototypes, numbers, array, null and any other edge case you could think of.

Reply:
Should be as noted in readme. but other cases would be nice